### ISSUE TITLE
Unable to run "main" branch with JDK 21: Caused by: java.lang.NoSuchMethodException: java.lang.Class.detach()

### ISSUE THREAD

**Issue by jluehe** [jiaqi/jmxterm#128]
Anyone else seeing this?
```
Exception seen:
cmdline = /[...]/bin/java --add-exports jdk.jconsole/sun.tools.jconsole=ALL-UNNAMED -jar jmxterm-1.0.5-uber.jar -l 62126 -v silent -i /tmp/jmxterm.tmp
Exception in thread "main" java.lang.RuntimeException: Can't cast sun.tools.attach.AttachProviderImpl@7f3b84b8: 62126 to VirtualMachineDescriptor
	at org.cyclopsgroup.jmxterm.jdk9.Jdk9JavaProcess.startManagementAgent(Jdk9JavaProcess.java:55)
	at org.cyclopsgroup.jmxterm.SyntaxUtils.getUrl(SyntaxUtils.java:46)
	at org.cyclopsgroup.jmxterm.boot.CliMain.execute(CliMain.java:135)
	at org.cyclopsgroup.jmxterm.boot.CliMain.main(CliMain.java:41)
Caused by: java.lang.NoSuchMethodException: java.lang.Class.detach()
	at java.base/java.lang.Class.getMethod(Class.java:2395)
	at org.cyclopsgroup.jmxterm.utils.WeakCastUtils.cast(WeakCastUtils.java:43)
	at org.cyclopsgroup.jmxterm.utils.WeakCastUtils.cast(WeakCastUtils.java:116)
	at org.cyclopsgroup.jmxterm.utils.WeakCastUtils.cast(WeakCastUtils.java:81)
	at org.cyclopsgroup.jmxterm.jdk9.Jdk9JavaProcess.startManagementAgent(Jdk9JavaProcess.java:52)
	... 3 more
```

**Comment by nyg** [comment]
Yes, it's a mistake in my previous PR. I created a new one, see #129. Thanks for the report!

**Comment by LeMyst** [comment]
> [@LeMyst](https://github.com/LeMyst) [@nyg](https://github.com/nyg), are there any plans to publish the `lemyst` releases to maven central or any other maven repo? Thank you!

https://github.com/LeMyst/jmxterm?tab=readme-ov-file#use-the-github-maven-repository

**Comment by jluehe** [comment]
> > [@LeMyst](https://github.com/LeMyst) [@nyg](https://github.com/nyg), are there any plans to publish the `lemyst` releases to maven central or any other maven repo? Thank you!
> 
> https://github.com/LeMyst/jmxterm?tab=readme-ov-file#use-the-github-maven-repository

Thanks so much, @LeMyst! Sorry I had missed the obvious :)

**Comment by LeMyst** [comment]
> > > [@LeMyst](https://github.com/LeMyst) [@nyg](https://github.com/nyg), are there any plans to publish the `lemyst` releases to maven central or any other maven repo? Thank you!
> > 
> > https://github.com/LeMyst/jmxterm?tab=readme-ov-file#use-the-github-maven-repository
> 
> Thanks so much, @LeMyst! Sorry I had missed the obvious :)

I just added it. Create an issue if this don't works.
